### PR TITLE
video: fix flaky beta tests

### PR DESCRIPTION
**Pull request by nnegrey**
Fix: https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3224
Fix: https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3223
Fix: https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3220

### CODE REVIEW

**Comment by nnegrey** [comment]
Different test is failing, which has been untouched so I need to go figure out why it now has an invalid argument. Possibly a change in the beta? Investigating.

**Comment by nnegrey** [comment]
Well turns out we don't even have tests for local files across the board. :O

**Comment by nnegrey** [comment]
I think the cat video is corrupted. As each time I use that video, it fails. But I tried another and it works...

File: video/cloud-client/analyze/analyze_test.py [hunk 1]
```diff
@@ -75,8 +82,29 @@ def test_detect_text_gcs(capsys):
...
 
 
+@pytest.mark.slow
+def test_detect_text(capsys):
+    analyze.video_detect_text("resources/googlework_tiny.mp4")
+    out, _ = capsys.readouterr()
+
+    text_exists = False
+    out_upper = out.upper()
+    for possible_text in POSSIBLE_TEXTS:
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply video/cloud-client/analyze/analyze_test.py:92]: Yea probably, I just copy-pasted the existing GCS ones and switched the method.

**Comment by tmatsuo** [comment]
Thanks @nnegrey for the fix. I'll merge once the tests pass

**Pull request merged**